README.md

```diff
@@ -35,7 +35,7 @@ we refer to
 
     Hutter, F. and Hoos, H. H. and Leyton-Brown, K.
     Sequential Model-Based Optimization for General Algorithm Configuration
-    In: Proceedings of the conference on Learning and Intelligent OptimizatioN (LION 5)
+    In: Proceedings of the conference on Learning and Intelligent Optimization (LION 5)
```

Comment:
The capital N is in fact intended, since it it LIO*N*

Reply:
oh, i even *tried* to look it up :D